source/slang/ir.cpp

```diff
@@ -4760,14 +4739,16 @@ namespace Slang
                             if (genericVal->op == kIROp_Func)
                             {
                                 auto genericFunc = (IRFunc*)genericVal;
-                                if (!genericFunc->genericDecl)
+                                if (!genericFunc->getGenericDecl())
                                     continue;
 
                                 // Okay, we have a candidate for specialization here.
                                 //
                                 // We will first find or construct a specialized version
                                 // of the callee funciton/
+                                auto originalFunc = dumpIRFunc(genericFunc);
```

Comment:
I assume this dumping logic shouldn’t be included, and it was just for debugging?